src/ResourceManager/Profile/Commands.Profile/Context/SaveAzureRMContext.cs

```diff
@@ -65,7 +67,7 @@ public override void ExecuteCmdlet()
                         throw new ArgumentException(Resources.AzureProfileMustNotBeNull);
                     }
 
-                    if (!AzureSession.DataStore.FileExists(Path) || Force ||
+                    if (!AzureSession.DataStore.FileExists(Path) || Force.IsPresent ||
```

Comment:
Why are we not using the same pattern as previous cmdlet
if (MyInvocation.BoundParameters.ContainsKey("Profile"))?
Also can we get away from writing verbose messages here?
Also any reason we are not executing WriteObject(ProfileObject) at the end?